src/containers/my-offers/my-offers-card/MyOffersCard.styles.ts

```diff
@@ -0,0 +1,49 @@
+import { TypographyVariantEnum } from '~/types'
+
+const ellipsisTextStyle = (linesCount: number) => ({
```

Comment:
we already use this function in another place, better move it out and reuse

Reply:
moved it out